src/jquery.mockjax.js

```diff
@@ -115,6 +115,15 @@
 				return null;
 			}
 		} else {
+
+			// Apply namespace prefix to the mock handler's url.
+			var namespace = handler.namespace || $.mockjaxSettings.namespace;
+			if (!!namespace) {
+				var namespacedUrl = [namespace, handler.url].join('/');
+				namespacedUrl = namespacedUrl.replace(/(\/+)/g, "/");
```

Comment:
Can we switch this to single quote style? I know... picky... but our build process runs jshint and _should_ fail on this. :)